classes/LocalizationPack.php

```diff
@@ -617,4 +618,26 @@ public function getErrors()
     {
         return $this->_errors;
     }
+
+    /**
+     * Return a currency (that can be soft deleted) if presents in the DB
+     * or a new one if not.
+     *
+     * @param string $isoCode
+     *
+     * @return Currency
+     *
+     * @throws PrestaShopDatabaseException
+     */
+    private function getSoftDeletedByIsoCodeOrCreate(string $isoCode): Currency
```

Comment:
looking at the method name, I would think that it will return an object only if it's soft-deleted or create a new one

Reply:
You're right, I'm gonna change that name 👍